### PR TITLE
fix: Enable all regions by default for managed app 

### PR DESCRIPTION
**Pull request by dciborow**

### CODE REVIEW

File: azureiai/managed_apps/confs/variant/feature_availability.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def set(
 
         if self.subtype == "ma":
             market_states = self.get_markets()
+            # Canada
+            market_states[22]["state"] = "Enabled"
```
Comment (dciborow) [inline azureiai/managed_apps/confs/variant/feature_availability.py:53]:
```suggestion
            market_states[24]["state"] = "Enabled"
```

File: azureiai/managed_apps/confs/variant/feature_availability.py [hunk 2]
```diff
@@ -60,9 +63,14 @@ def set(
                 "priceSchedules": [
                     {
                         "isBaseSchedule": False,
-                        "marketCodes": ["US"],
+                        "marketCodes": ["CA", "US"],
                         "friendlyName": "free_priceOverrideSchedule_US",
                         "schedules": [
+                            {
+                                "retailPrice": {"openPrice": 0, "currencyCode": "USD"},
```
Comment (manuvar-ms) [inline azureiai/managed_apps/confs/variant/feature_availability.py:70]: Would this support subscriptions with currency set to CAD (Canadian dollars)? My subscription is set to use CAD.

File: azureiai/managed_apps/confs/variant/feature_availability.py [hunk 3]
```diff
@@ -49,6 +49,9 @@ def set(
 
         if self.subtype == "ma":
             market_states = self.get_markets()
+            # Canada
+            market_states[24]["state"] = "Enabled"
```
Comment (dciborow) [inline azureiai/managed_apps/confs/variant/feature_availability.py:53]:
```suggestion
            market_states[23]["state"] = "Enabled"
```

File: azureiai/managed_apps/confs/variant/feature_availability.py (hunk 2 above)
Comment (manuvar-ms) [reply azureiai/managed_apps/confs/variant/feature_availability.py:73]: I can test with USD to start with.

**Pull request merged**